src/main/java/com/codingParkFun/rmsbackend/controller/StudentController.java

```diff
@@ -28,5 +30,32 @@ public void createStudent( @RequestBody Student student ) {
         return studentService.get();
     }
 
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public Optional<Student> getStudent( @PathVariable String id ) {
+        if (id == null) {
+            throw new IdDoesNotExistException( "Student does not exist with this id" );
+        }
+        long studentId = Long.parseLong(id.substring( 1,id.length()-1 ));
+        return studentService.getById( studentId );
+    }
+
+    @DeleteMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public void deleteStudent( @PathVariable String id ) {
+        if (id == null) {
+            throw new IdDoesNotExistException( "Student does not exist with this id" );
+        }
+        long studentId = Long.parseLong(id.substring( 1,id.length()-1 ));
+        studentService.delete( studentId );
+    }
+
+    @DeleteMapping
+    @ResponseStatus(HttpStatus.OK)
+    public void deleteAllStudents(){
+        studentService.deleteAll();
```

Comment:
This will delete all the students from all the schools. We can't afford that.

Reply:
ok.  i will remove that.

Reply:
There should be an API to remove all the students for a class.